### PR TITLE
Master

### PR DESCRIPTION
**Pull request by chrisgriffith**
Added a reference to the text-left and text-right on ion-checkbox,
ion-select, and ion-toggle components.

### CODE REVIEW

File: content/_includes/v2_fluid/component-docs/toolbar.html [hunk 1]
```diff
@@ -50,7 +50,7 @@ <h3 class="no-para">Header</h3>
   <ion-toolbar color="primary">
     <ion-buttons start>
       <button ion-button icon-only>
-        <ion-icon name="content"></ion-icon>
+        <ion-icon name="more"></ion-icon>
```
Comment (mhartington) [inline content/_includes/v2_fluid/component-docs/toolbar.html]: This has already been updated and is not needed.

File: content/docs/v2/theming/css-utilities/index.md [hunk 2]
```diff
@@ -66,9 +66,15 @@ Ionic provides a set of SCSS/CSS utility attributes that can help when developin
...
 |-----------|------------------------------------------------------------------------------------------------------------|
-| `start`   | Aligns the element at the start based on platform. iOS goes left, Android is the first item on the right   |
-| `end`     | Aligns the element at the end based on the platform. iOS goes right, Android is the last item on the right |
-| `left`    | Aligns the element on the left side. Overrides the platform specific order                                 |
-| `right`   | Aligns the element on the right side. Overrides the platform specific order                                |
+| `start`   | aligns the element at the start based on platform. iOS goes left, Android is the first item on the right   |
+| `end`     | aligns the element at the end based on the platform. iOS goes right, Android is the last item on the right |
+| `left`    | aligns the element on the left side. Overrides the platform specific order                                 |
+| `right`   | aligns the element on the right side. Overrides the platform specific order                                |
```
Comment (mhartington) [inline content/docs/v2/theming/css-utilities/index.md]: Please fix this casing.

File: content/docs/v2/theming/css-utilities/index.md [hunk 3]
```diff
@@ -66,9 +66,15 @@ Ionic provides a set of SCSS/CSS utility attributes that can help when developin
...
 
 
+### `ion-checkbox, ion-select, ion-toggle` property
```
Comment (mhartington) [inline content/docs/v2/theming/css-utilities/index.md]: Isn't this already covered by 
https://github.com/driftyco/ionic-site/blob/master/content/docs/v2/theming/css-utilities/index.md#text-transformation

If not, we could just add a note there instead to cut down on repeat content.

**Comment by chrisgriffith** [comment]
Changed the cases for all the descriptions to upper case. Made an initial mistake, the correct attribute is item-right or item-left. Thus the text-transformation reference would not be correct.

**Pull request closed without merging**